authz/providers/azure/options/options.go

```diff
@@ -47,6 +47,7 @@ type Options struct {
 	AllowNonResDiscoveryPathAccess      bool
 	UseNamespaceResourceScopeFormat     bool
 	DiscoverResources                   bool
+	EnableManagedNamespaceRBAC          bool
```

Comment:
could we UseManagedNamespaceResourceScopeFormat to follow the existing variable pattern (UseNamespaceResourceScopeFormat)

Reply:
also could we also initialize this new variable to false in NewOptions() at Ln 62